lstchain/scripts/onsite/onsite_create_cat_B_calibration_files_with_batch.py

```diff
@@ -0,0 +1,197 @@
+#!/usr/bin/env python
+
+"""
+
+ Onsite script to process (in batch) the interleaved events of several runs 
+ and to create the Cat-B calibration files
+
+ --> onsite_create_catB_calibration_files_with_batch -r xxx yyy zzz
+
+"""
+
+import argparse
+import shutil
+import subprocess
+import sys
+from datetime import datetime
+from pathlib import Path
+
+import lstchain
+from lstchain.onsite import (
+    DEFAULT_BASE_PATH,
+    CAT_B_PIXEL_DIR,
+    find_interleaved_subruns,
+    DEFAULT_CONFIG_CAT_B_CALIB,
+)
+
+# parse arguments
+parser = argparse.ArgumentParser(
+    description='Reconstruct filter scan, this must be run after the night calibration scripts',
+    formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+required = parser.add_argument_group('required arguments')
+optional = parser.add_argument_group('optional arguments')
+
+required.add_argument('-r', '--run_list', help="Run numbers of intereleaved data",
+                      type=int, nargs="+")
+optional.add_argument('-f', '--filters_list', help="Filter list (same order as run list)",
```

Comment:
isn't is possible to get this list directly from the DB?

Reply:
This is for the case in which it is not possible, as for the Cat-A